brand-guidelines/typography.md

```diff
@@ -28,7 +28,14 @@ A clear typographic hierarchy is critical to the effective communication of our
 
 </div>
 
-<h2 id="typefaces">Typefaces<span class="cf-code-link"><a href="https://cfpb.github.io/cf-typography/docs/">View code <span class="cf-icon cf-icon-external-link"></span></a></span></h2>
+<h2 id="typefaces">
+    Typefaces
+    <span class="cf-code-link">
+        <a href="https://github.com/cfpb/capital-framework/tree/canary/src/cf-typography/cf-typography">
```

Comment:
Hrm... should we link this to cf-core, which really does the base typographic styling?